packages/twenty-server/src/engine/metadata-modules/field-metadata/utils/unserialize-default-value.ts

```diff
@@ -0,0 +1,42 @@
+import { FieldMetadataDefaultSerializableValue } from 'src/engine/metadata-modules/field-metadata/interfaces/field-metadata-default-value.interface';
+
+export const unserializeDefaultValue = (
```

Comment:
introducing this function to have a robust way to unserizalied (remove quotes) in defaultValues